File: Python/UberVan.py
from car import Car


class UberVan (Car):
    typerCarAccepter = []
    seatsMaterial    = []
    
    
    def __init__(self,license, driver, typerCarAccepter, seatsMaterial):
        super().__init__(license, driver)
        self.typerCarAccepter = typerCarAccepter
        self.seatsMaterial = seatsMaterial

File: java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

class UberVan  extends Car{
    Map<String,  Map<String, Integer>> typerCarAccepter;
    ArrayList<String> seatsMaterial;



    public UberVan (String license, Account driver,Map<String,  Map<String, Integer>> typerCarAccepter,
    ArrayList<String> seatsMaterial){
        super(license, driver);
        this.typerCarAccepter = typerCarAccepter;
        this.seatsMaterial = seatsMaterial;

    }
}        